documentation/API-onboarding.md

```diff
@@ -43,8 +43,8 @@ In parallel, the API proposal is formally registered in CAMARA by adding it to t
 
 Upon receiving the API proposal and notification from the WG chair, the TSC studies the proposal and votes it at the TSC meeting.
 
-- No-Go! Objecting companies shall provide justifications why. The API proposal is rejected and will not be included in any API Sub Project.
-- Go! In this case, the TSC shall specify whether the API proposal is to be hosted by a new or existing Sub Project.
+- **Not Accepted**: The API proposal is rejected and will not be included in any API Sub Project, and shall either (1) be removed from the API Backlog, with objecting companies to provide justifications why, or (2) returned to the API Backlog, with the TSC including in the 'justications' section the  clarifications required/gaps to be addressed/corrections required in order for the API to be re-submitted by the API supporters.
```

Comment:
@Kevsy: I support the proposal done by @jordonezlucena in https://github.com/camaraproject/Governance/pull/88#discussion_r1321853248:

```suggestion
- **Not Accepted**: The API proposal is rejected, and thus will not be included in any API Sub-Project. The TSC will need to inform of this decision to the API backlog WG, and clarify next steps: either (1) remove API proposal from backlog, with objecting companies providing justifications why; or (2) ask for changes (e.g., clarifications, corrections, gaps to be addressed) required in order for the API to be re-submitted by the API supporters.
```

One reason is that I wouldn't know where his 'justification' section is and it looks clearer for me if the API Backlog WG drives the process based on the TSC decision.